.github/workflows/ci_flutter.yml

```diff
@@ -0,0 +1,53 @@
+name: Build flutter example
+
+on:
+  workflow_call:
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
```

Comment:
I think we should remove this until they update the documentation that explains how this works in a reusable workflow. Also, we already define a concurrency group in the calling workflow, so that workflow will already be canceled when a new version is pushed